test/test_integration_cluster.rb

```diff
@@ -12,7 +12,25 @@ def setup
   end
 
   def teardown
-    super if HAS_FORK
+    return if skipped?
+    super
+  end
+
+  def test_pre_existing_unix
+    skip UNIX_SKT_MSG unless UNIX_SKT_EXIST
+
+    File.open(@bind_path, mode: 'wb') { |f| f.puts 'pre eixisting' }
+
+    cli_server "-w #{WORKERS} -t 0:6 -q test/rackup/sleep_step.ru", unix: :unix
```

Comment:
leave out the thread setting, not necessary

Reply:
Done